model_compression_toolkit/core/common/quantization/quantization_fn_selection.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2021 Sony Semiconductor Israel, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+from collections.abc import Callable
+from functools import partial
+
+from model_compression_toolkit.core.common.target_platform import QuantizationMethod
+from model_compression_toolkit.core.common.quantization.quantizers.kmeans_quantizer import kmeans_quantizer
+from model_compression_toolkit.core.common.quantization.quantizers.lut_kmeans_quantizer import lut_kmeans_quantizer
+from model_compression_toolkit.core.common.quantization.quantizers.uniform_quantizers import power_of_two_quantizer, \
+    symmetric_quantizer, uniform_quantizer
+
+
```

Comment:
I don't like the name "selection" here.
It can be confusing with "parameters selection" which uses "QuantizationErrorMethod" enum (too many things that are too similar and it can create confusion).
Consder: "get_quantization_fn" or "quantization_fn_generator" or something like that.